addons/account_peppol/models/account_edi_proxy_user.py

```diff
@@ -76,6 +76,51 @@ def _get_proxy_identification(self, company, proxy_type):
             return f'{company.peppol_eas}:{company.peppol_endpoint}'
         return super()._get_proxy_identification(company, proxy_type)
 
+    def _peppol_save_new_document(self, attachment, company, partner_endpoint, peppol_state, uuid):
+        """Save new documents in an accounting journal, when one is specified on the company.
+
+        :param attachment: the new document
+        :param company: the company where we want to save the document
+        :param partner_endpoint: a string containing the sender's Peppol endpoint
+        :param peppol_state: the state of the received Peppol document
+        :param uuid: the UUID of the Peppol document
+        :return: `True` if the document was saved, `False` if it was not
+        """
+        journal_id = company.peppol_purchase_journal_id
+        if not journal_id:
+            return False
+
+        try:
+            move = journal_id\
+                .with_context(
+                    default_move_type='in_invoice',
+                    default_peppol_move_state=peppol_state,
+                    default_extract_can_show_send_button=False,
+                    default_peppol_message_uuid=uuid,
+                )\
+                ._create_document_from_attachment(attachment.id)
+            if partner_endpoint:
+                move._message_log(body=_(
+                    'Peppol document has been received successfully. Sender endpoint: %s', partner_endpoint))
+            else:
+                move._message_log(body=_('Peppol document has been received successfully'))
```

Comment:
```suggestion
            message_body = 'Peppol document has been received successfully.'
            if partner_endpoint:
                message_body += ' Sender endpoint: %s' % partner_endpoint
            
            move._message_log(body=message_body)
```
Same logic as comment on the enterprise PR, it avoid the repetition of base sentence